testSetup.js

```diff
@@ -1,4 +1,5 @@
 /* eslint prefer-template: 0 */
+import 'jest-enzyme';
```

Comment:
Try to get latest master or this commit: 4af134583be5eab3adb898f879879c25fa7ee517